File: fibonacci/fib.js
function fib(n) {
  if (n < 2) {
    return n;
  }

  return fib(n - 1) + fib(n - 2);
}

const startMs = new Date();
fib(40);
const endMs = new Date();

console.log(`EXECUTION TIME: ${endMs - startMs} ms`);

File: fibonacci/fib.c
#include <stdio.h>
#include <time.h>
#include <math.h>

long millis(){
    struct timespec _t;
    clock_gettime(CLOCK_REALTIME, &_t);
    return _t.tv_sec*1000 + lround(_t.tv_nsec/1.0e6);
}

int fibonacci(int n) {
  if (n < 2) {
    return n;
  }

  return fibonacci(n - 1) + fibonacci(n - 2);
}

int main() {
  long startMs = millis();
  fibonacci(40);
  long endMs = millis();

  printf("EXECUTION TIME: %lu ms\n", (endMs - startMs));
  return 0;
}
